BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -73,6 +65,10 @@ protected async void OnEnable()
                 TrackableId trackableId = m_anchorStore.LoadAnchor(name);
                 m_incomingPersistedAnchors.Add(trackableId, name);
             }
+
+            // Anchors may already exist in the subsystem, but are not guaranteed to be loaded into the scene until
+            // the AnchorsChanged event. Setting this flag tells this app to process existing anchors at that time.
+            m_processExistingAnchorsOnAnchorsChanged = true;
```

Comment:
I dont think we need to set a flag in OnEnable(). Is it not given that AnchorsChanged() gets called only after OnEnable()? Instead, we can have something like "m_processedExistingAnchors" be set once the anchors are processed on the first AnchorsChanged() event.

Reply:
I think the real question is whether or not we care to support the scenarios around this being enabled / disabled, as a more robust sample, even if we're not using it that way.

This script could definitely be simplified if we make assumptions like what you suggested, but I've been leaning towards making this a more robust example of best practices, for general use

Reply:
To support the flag like you mentioned, we'd still need to reset the flag on disable or enable to handle the case where this is enabled -> disabled -> enabled